app/managers/session_manager.ts

```diff
@@ -7,6 +7,7 @@ import FastImage from 'react-native-fast-image';
 
 import {removePushDisabledInServerAcknowledged, storeOnboardingViewedValue} from '@actions/app/global';
 import {cancelSessionNotification, logout, scheduleSessionNotification} from '@actions/remote/session';
+import {callsOnAppStateChange} from '@calls/state';
```

Comment:
does it really make sense to have this in the session manager? perhaps it makes more sense to have it in `init/push_notifications` as is notifications related or perhaps in the websocket manager?

This also brings up the question as we do listen for that event in multiple places, would it make sense to have a new file `AppStateManager` to handle app state changes in one central place? thoughts? cc: @larkox  (obviously that may be better to have in it's own PR) <- if we go this route, no need to change this here.

Reply:
Makes sense -- I made our own CallsManager, to keep it clean and clear what's going on. (The reason I put it in session manager and not websocket manager is that we only need it set up once, not for every server.)